src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create driver. " + driver + " ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT * FROM drivers"
+                + " WHERE driver_id = (?) AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            Driver driver = null;
+            if (resultSet.next()) {
+                driver = getDriver(resultSet);
+            }
+            return Optional.ofNullable(driver);
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get driver by id " + id + " ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String query = "SELECT * FROM drivers WHERE is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            List<Driver> drivers = new ArrayList<>();
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get a list of drivers "
+                    + "from drivers table. ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String query = "UPDATE drivers SET driver_name = ?, driver_license_number = ?"
+                + " WHERE driver_id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setString(1, driver.getName());
+            preparedStatement.setString(2, driver.getLicenseNumber());
+            preparedStatement.setLong(3, driver.getId());
+            preparedStatement.executeUpdate();
+            return driver;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't update a driver "
+                    + driver + " ", throwable);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE drivers SET is_deleted = TRUE WHERE driver_id = ?";
```

Comment:
```suggestion
        String query = "UPDATE drivers SET is_deleted = TRUE WHERE id = ?";
```
same for other naming

Reply:
done